lib/rules/common/tap-synonyms.js

```diff
@@ -0,0 +1,87 @@
+'use strict'
+// @see https://github.com/tapjs/node-tap/blob/f323cdca28cc0a286e29585d8081e04b08704143/lib/synonyms.js
+// A list of all the synonyms of assert methods.
+// In addition to these, multi-word camelCase are also synonymized to
+// all lowercase and snake_case
+
+function multiword(obj) {
+  return Object.keys(obj).reduce((s, i) => {
+    s[i] = [multiword_(i)]
+      .concat(obj[i].map(multiword_))
+      .reduce((s, i) => {
+        s.push.apply(s, i)
+        return s
+      }, [])
+    return s
+  }, obj)
+}
+
+function multiword_(str) {
+  return str.match(/[A-Z]/) ? [
+    str
+  , str.toLowerCase()
+  , str.replace(/[A-Z]/g, $0 => '_' + $0.toLowerCase())
+  ] : [str]
+}
+
+module.exports = multiword({
+  'ok': ['true', 'assert']
+, 'notOk': ['false', 'assertNot']
+
+, 'error': ['ifError', 'ifErr']
+, 'throws': ['throw']
+, 'doesNotThrow': ['notThrow']
+
+  // exactly the same.  ===
+, 'equal': [
+    'equals', 'isEqual', 'is', 'strictEqual', 'strictEquals', 'strictIs'
+  , 'isStrict', 'isStrictly', 'identical'
+  ]
+
+  // not equal.  !==
+, 'not': [
+    'inequal', 'notEqual', 'notEquals', 'notStrictEqual', 'notStrictEquals'
+  , 'isNotEqual', 'isNot', 'doesNotEqual', 'isInequal'
+  ]
+
+  // deep equivalence.  == for scalars
+, 'same': [
```

Comment:
I think the short names like `equal` and `same` and a little ambiguous and cause some confusion. 

I think we should use the `strictX` and `deepX` variants where available.

Reply:
nvm. It is 😄